packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -398,4 +422,4 @@ - (NSString *)navigateAtEnd
 	return @"NavigateStopAtEnds";
```

Comment:
That was not something we (the FURN macOS crew) wrote.. we basically inherited the whole macOS RCTFocusZone implementation from a partner team who wrote it themselves. We elected to take their implementation and open source it, without changing too much. That is why, for example, this is _RCT_FocusZone instead of _FRN_FocusZone (FURN Objective C components shouldn't prefix RCT because of namespace issues)